engine/engine/src/engine_main.cpp

```diff
@@ -38,6 +40,8 @@ static void AppDestroy(void* _ctx)
 
 static int EngineMain(int argc, char *argv[])
 {
+    dmExportedSymbols(); // Instead of our previous global constructor
```

Comment:
EngineMain is the first common entry point, as each platform (and extender) behave a little differently.